src/toast/ops/stokes_weights/kernels_jax.py

```diff
@@ -116,17 +191,18 @@ def stokes_weights_IQU_interval(
         quats, (quat_index, intervals, ALL)
     )  # quats[quat_index,intervals,:]
 
-    # insures hwp is a non empty array
+    IAU_sign = 1.0 - 2.0 * IAU
```

Comment:
This could use the [sign function](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.sign.html).

Reply:
(or a one line test if IAU is static, something like: `IAU_sign = 1 if IAU else -1`)